app/helpers/application_helper.rb

```diff
@@ -15,7 +15,7 @@ def generate_history_text url
       puts "#"*20
       puts e.message
       puts "#"*20
-      return url
+      return "/" # default to home page.
```

Comment:
I believe if it correctly ignores your shib then it shouldn’t be as big of an issue.

From: <owner-sparc_request@musc.edu<mailto:owner-sparc_request@musc.edu>> on behalf of robertjamesmiller <notifications@github.com<mailto:notifications@github.com>>
Reply-To: sparc-request/sparc-fulfillment <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, December 10, 2015 at 12:46 PM
To: sparc-request/sparc-fulfillment <sparc-fulfillment@noreply.github.com<mailto:sparc-fulfillment@noreply.github.com>>
Subject: Re: [sparc-fulfillment] The Iowa Shibboleth referrer is /idp/profile/SAML2/Redirect/SSO?back=true (#102)




In app/helpers/application_helper.rb<https://github.com/sparc-request/sparc-fulfillment/pull/102#discussion_r47258658>:

> @@ -15,7 +15,7 @@ def generate_history_text url
>        puts "#"*20
>        puts e.message
>        puts "#"*20
> -      return url
> +      return "/" # default to home page.


I updated 'return "/"' because of the goofy mouseover "title" that was being displayed after a user had just logged in with Shibboleth, but mouseover titles aren't that important so I can take it out.

—
Reply to this email directly or view it on GitHub<https://github.com/sparc-request/sparc-fulfillment/pull/102/files#r47258658>.